### PR TITLE
fix(SUP-38407): [GaTech] [Multistream - V7 Player] Child entries not loading high quality flavor assets

### PR DESCRIPTION
**Pull request by Tzipi-kaltura**
Issue:
Quality is not changed in childe entries when it is changed in the parent entry.

**Fix:**
Add listener to videotrackchange and change the quality manually to all the other entries.

part of - https://github.com/kaltura/playkit-js/pull/793, https://github.com/kaltura/kaltura-player-js/pull/851

solves [SUP-38407](https://kaltura.atlassian.net/browse/SUP-38407)

[SUP-38407]: https://kaltura.atlassian.net/browse/SUP-38407?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/dualscreen.tsx [hunk 1]
```diff
@@ -201,6 +201,9 @@ export class DualScreen extends BasePlugin<DualScreenConfig> implements IEngineD
         }
       }
     });
+    this.eventManager.listen(this.player, EventType.VIDEO_TRACK_CHANGED, (event) => {
```
Comment (semarche-kaltura) [inline src/dualscreen.tsx:204]: do we need remove listener on reset?

File: src/dualscreen.tsx [hunk 2]
```diff
@@ -230,6 +233,20 @@ export class DualScreen extends BasePlugin<DualScreenConfig> implements IEngineD
     return thumbs[0] && thumbs[1] ? thumbs : thumbs[0] ?? thumbs[1];
   };
 
+  private _changeQuality = (label: string) => {
+    this._dualScreenPlayers.forEach(dualScreenPlayer => {
+      if(dualScreenPlayer.id != 'mainPlayer' && dualScreenPlayer.id != 'imagePlayer'){
```
Comment (semarche-kaltura) [inline src/dualscreen.tsx:238]: 1. use prettier.
there should be space between `if` and `(`, same between `)` and `{`
2. use strict compare `!==` that requires less resources
3. use enums
`IMAGE_PLAYER_ID` for 'imagePlayer'
`MAIN_PLAYER_ID` for 'mainPlayer'

**Review by semarche-kaltura — CHANGES_REQUESTED** [review]

File: src/dualscreen.tsx [hunk 3]
```diff
@@ -230,6 +233,20 @@ export class DualScreen extends BasePlugin<DualScreenConfig> implements IEngineD
     return thumbs[0] && thumbs[1] ? thumbs : thumbs[0] ?? thumbs[1];
   };
 
+  private _changeQuality = (label: string) => {
+    this._dualScreenPlayers.forEach(dualScreenPlayer => {
+      if (dualScreenPlayer.id !== MAIN_PLAYER_ID && dualScreenPlayer.id !== IMAGE_PLAYER_ID) {
```
Comment (semarche-kaltura) [inline src/dualscreen.tsx:238]: I would do
```
![MAIN_PLAYER_ID, IMAGE_PLAYER_ID].includes(dualScreenPlayer.id)
```
but thats only syntax, your changes looks good!

**Review by semarche-kaltura — APPROVED** [review]

File: src/dualscreen.tsx [hunk 4]
```diff
@@ -230,6 +233,20 @@ export class DualScreen extends BasePlugin<DualScreenConfig> implements IEngineD
...
+        //@ts-expect-error
+        const videoTrack = dualScreenPlayer.player.getTracks('video');
+        const selectedVideoTrack = videoTrack.find((track: any) => track.label === label);
+        if (selectedVideoTrack) {
+          //@ts-expect-error
+          dualScreenPlayer.player.changeChildQuality(selectedVideoTrack);
```
Comment (MosheMaorKaltura) [inline src/dualscreen.tsx:244]: rename to changeQuality

Comment (Tzipi-kaltura) [reply src/dualscreen.tsx]: done

**Review by MosheMaorKaltura — APPROVED** [review]

**Pull request merged**